patches/extra/ungoogled-chromium/enable-paste-and-go-new-tab-button.patch

```diff
@@ -0,0 +1,34 @@
+--- a/chrome/browser/ui/views/tabs/new_tab_button.cc
++++ b/chrome/browser/ui/views/tabs/new_tab_button.cc
+@@ -63,10 +63,8 @@ const gfx::Size NewTabButton::kButtonSize{28, 28};
+ NewTabButton::NewTabButton(TabStrip* tab_strip, views::ButtonListener* listener)
+     : views::ImageButton(listener), tab_strip_(tab_strip) {
+   set_animate_on_state_change(true);
+-#if defined(OS_LINUX) && !defined(OS_CHROMEOS)
+   set_triggerable_event_flags(triggerable_event_flags() |
+                               ui::EF_MIDDLE_MOUSE_BUTTON);
+-#endif
+ 
+   // Initialize the ink drop mode for a ripple highlight on button press.
+   ink_drop_container_ = new views::InkDropContainerView();
+--- a/chrome/browser/ui/views/tabs/tab_strip.cc
++++ b/chrome/browser/ui/views/tabs/tab_strip.cc
+@@ -2699,15 +2699,15 @@ void TabStrip::ButtonPressed(views::Button* sender, const ui::Event& event) {
+     if (event.IsMouseEvent()) {
+       const ui::MouseEvent& mouse = static_cast<const ui::MouseEvent&>(event);
+       if (mouse.IsOnlyMiddleMouseButton()) {
+-        if (ui::Clipboard::IsSupportedClipboardType(
+-                ui::CLIPBOARD_TYPE_SELECTION)) {
++        //if (ui::Clipboard::IsSupportedClipboardType(
++                //ui::CLIPBOARD_TYPE_SELECTION)) {
```

Comment:
It's not really clear from this patch alone why this needs to be commented out. Could you elaborate on this?

Reply:
https://github.com/chromium/chromium/blob/2d57e5b8afc6d01b344a8d95d3470d46b35845c5/ui/base/clipboard/clipboard.h#L40-L50

Here is `IsSupportedClipboardType` method, that always returns `false` when type is `CLIPBOARD_TYPE_SELECTION` and platform is non-Linux. So I hided this condition.
I thought that changing something in this method is quite dangerous, because something can break in a completely different place.

Reply:
Hmm I see. The reason I asked about this is [Line 27 (of the patch file itself)](https://github.com/Eloston/ungoogled-chromium/pull/742/files#diff-44c235527014ca5a4e29a16670d058f6R27) uses `CLIPBOARD_TYPE_SELECTION`, so I was wondering if this could cause any problems on non-Linux platforms.

Reply:
It works fine on OS X.
Unfortunately, windows build is broken and I can't test it there. =(

But it always has condition for non-empty text from clipboard.

Reply:
I see. As long as the clipboard always exists (even if it only returns empty strings), then that works for me. Thanks.